### PR TITLE
Tweak default histogram buckets

### PR DESCRIPTION
**Pull request by jvff**
## Motivation

The default histogram buckets don't map particularly well to some metrics. Finding optimal values will require some discussion and experimentation. However, we could use better temporary values until then.

## Proposal

Tweak the buckets of some histogram metrics, to use temporary values that are better than the defaults.

## Test Plan

Manually executed the Crowd-Funding integration tests tweaked to print out the metrics in the end.

## Release Plan

Internal changes, nothing needed.

## Links

- [reviewer checklist](https://github.com/linera-io/linera-protocol/blob/main/CONTRIBUTING.md#reviewer-checklist)

### CODE REVIEW

**Review by andresilva91 — APPROVED** [review]
Nice! Thank you for adjusting these 🙏🏻
[nicesteve](https://app.graphite.dev/macros/linera-io)

File: linera-chain/src/chain.rs
Comment (andresilva91) [inline linera-chain/src/chain.rs:58]: One thing is that once #1523 merges these will likely have to be adjusted (they'll be in ms now instead of seconds)

Comment (jvff) [reply linera-chain/src/chain.rs:58]: Got it. Should I wait for it to be merged or merge this one first?

**Pull request merged**
